lib/platforms/ios.dart

```diff
@@ -126,7 +126,7 @@ void _setIOSPackageName(dynamic packageName) {
         // `PRODUCT_BUNDLE_IDENTIFIER = "{{BUNDLE_ID}}.{{EXTENSION_NAME}}";`
         .replaceAllMapped(
       RegExp(
-        r'PRODUCT_BUNDLE_IDENTIFIER = "([A-Za-z0-9.-]+)\.([A-Za-z0-9.-]+)";',
+        r'PRODUCT_BUNDLE_IDENTIFIER = ([A-Za-z0-9.-_]+)\.([A-Za-z0-9.-_]+);',
```

Comment:
Why did you remove the `"` ? The extension PRODUCT_BUNDLE_IDENTIFIER have those, if removed, the regex who also match non expected elements.

Reply:
@OutdatedGuy reverted back double qoutes, not sure why i have removed that. Since regex works with them as well